src/components/Navigation.astro

```diff
@@ -1,10 +1,16 @@
 ---
 import NavigationLink from './NavigationLink.astro'
-import { faCode, faFloppyDiskPen, faWrench } from '@fortawesome/sharp-solid-svg-icons'
 ---
 
-<>
-  <NavigationLink label="Blog" href="/blog" icon={faFloppyDiskPen} />
-  <NavigationLink label="Snippets" href="/snippets" icon={faCode} />
-  <NavigationLink label="Projects" href="/projects" icon={faWrench} />
-</>
+<section class="flex items-center flex-col md:flex-row gap-4 py-4 md:py-8">
+  <div class="flex items-center">
+    <NavigationLink label="About" href="/about" />
```

Comment:
These values should be stored in an array and rendered accordingly. (DRY - **D**on't **R**epeat **Y**ourself)

Reply:
Remainder, all values that are exported and generally represent static values should be always written in UPPERCASE, with _ and be stored in a `.consts.ts` file. 

Example: `NAVIGATION_LINKS` - you could still name hooks, dynamic values etc (inside of functions and components) and use camelCase, but for everything else - as mentioned above - we should follow thee principle.